resources/bundles/org.eclipse.core.resources/src/org/eclipse/core/internal/localstore/FileSystemResourceManager.java

```diff
@@ -903,6 +903,38 @@ public InputStream read(IFile target, boolean force, IProgressMonitor monitor) t
 		}
 	}
 
+	/** @see #read(IFile, boolean, IProgressMonitor) **/
+	public byte[] readAllBytes(IFile target, boolean force, IProgressMonitor monitor) throws CoreException {
```

Comment:
Must this be public or could it be package private maybe?

Reply:
must be public otherwise org.eclipse.core.internal.resources.File.readAllBytes() -> "The method readAllBytes(IFile, boolean, IProgressMonitor) from the type FileSystemResourceManager is not visible"